lib/dataElementListener.js

```diff
@@ -372,6 +372,14 @@ class DataElementImporter extends SHRDataElementParserListener {
       } else if (cst.elementStringConstraint()) {
         const str = stripDelimitersFromToken(cst.elementStringConstraint().STRING());
         value.addConstraint(new FixedValueConstraint(str, 'string', path));
+      } else if (cst.elementIntegerConstraint()) {
+        const minus = cst.elementIntegerConstraint().MINUS() ? '-' : '';
+        const num = cst.elementIntegerConstraint().WHOLE_NUMBER().getText();
+        const int = parseInt(minus + num);
```

Comment:
Is there a reason why you don't take the same approach as you do for decimal? e.g.,
```
const int = parseInt(cst.elementIntegerConstraint().getText().substr(1));
```
Both approaches are correct, but I think consistency would be good: either process the MINUS independently or not (for both int and decimal).